README.md

```diff
@@ -14,8 +14,49 @@ You can try the [live demo ↗](http://hugogiraudel.github.io/a11y-dialog/exampl
 
 ## Installation
 
+### Using a bundler (recommended)
+
+If you’re using a bundler (such as Webpack or Rollup), you can install `a11y-dialog` through npm or yarn like any other dependency:
+
+```sh
+npm install a11y-dialog
+yarn add a11y-dialog
```

Comment:
I tend not to include both npm and yarn commands in case people copy-paste, or for begginers who might type it out as written in the docs.

One solution is to split it:

    ```sh
    npm install a11y-dialog
    ```

    or with Yarn:

    ```sh
    yarn add a11y-dialog
    ```

Another one is to use a shell comment:

    ```sh
    npm install a11y-dialog  # or: yarn add a11y-dialog
    ```

Personally, I tend to only give the npm command, since it’s the default tool that comes with Node.js, but it's also my personal bias since I don’t use Yarn much.